diff --git a/config.def.h b/config.def.h
index f72ff62..63abab2 100644
--- a/config.def.h
+++ b/config.def.h
@@ -63,7 +63,7 @@ static const Layout layouts[] = {
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *termcmd[] = {"tabbed", "-c", "-r", "2", "st", "-w", "''", NULL};
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
