diff --git a/config.def.h b/config.def.h
index e06be66..4478cb2 100644
--- a/config.def.h
+++ b/config.def.h
@@ -63,5 +63,10 @@ static const char unknown_str[] = "n/a";
  */
 static const struct arg args[] = {
 	/* function format          argument */
+	{ battery_perc, "BAT0"},
 	{ datetime, "%s",           "%F %T" },
+	{ wifi_essid, "wlp4s0"},
+	{ username, NULL },
+	{ vol_perc, "/dev/mixer"},
+	{ uptime, NULL },
 };
